components/user-mgt/org.wso2.carbon.identity.user.profile.ui/src/main/resources/web/userprofile/edit.jsp

```diff
@@ -85,7 +85,7 @@
 %>
 <script type="text/javascript">
     function forward() {
-        location.href = "<%=forwardTo%>";
+        location.href = "<%=Encode.forJavaScriptBlock(forwardTo)%>";
```

Comment:
forwardTo is hardcoded and does not contain untrusted data. Not necessary to encode